### ISSUE TITLE
Monk leaden fist tracker?

### ISSUE THREAD

**Issue by HydrophobicWalrus** [0ceal0t/JobBars#21]
A small ask but one that would be nice as quality of life, would it be possible to have a proc icon to indicate if you have a leaden fist buff active?

**Comment by 0ceal0t** [comment]
I personally don't really see this as useful, and I'd like to prevent gauge bloat if possible.

I'm by no means a MNK main, but I understand that you just alternate dragon kick > bootshine > dragon kick > ... every 3 gcds (or every other in perfect balance). Keeping track of that without a gauge seems very simple, so I'm unclear about the need for this.

**Comment by 0ceal0t** [comment]
of course, feel free to correct me if I'm missing something. like I said, I don't really play MNK at all

**Comment by HydrophobicWalrus** [comment]
I was more just thinking one dot thing like you have for charges of true north/riddle. I'm still fairly new with monk at high end and with my terrible short memory I find it hard sometimes to remember if I'm due to boot/dragon. It's probably just a me issue but I feel like it's a nice small feature

**Comment by Thalassocrat** [comment]
This is a feature I would be most looking for on MNK and wanted to post for, personally. Was sad to see it's not here and neither is it in Remind Me. There are so many buffs on your bar when playing MNK and the positioning of Leaden Fist on it isn't always the same, sometimes I lose track and it's inconvenient to scan the buff bar. An icon in a gauge like this would be a lot more useful than you say, for me. Each section is already toggleable on or off... So here's my vote in favor of that.

**Comment by mkaito** [comment]
If you're just chilling doing your rotation, you wouldn't need it. But if you get interrupted by some mechanic, it'd be useful to quickly double check where you're at, I suppose.
